extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -721,7 +721,8 @@ export class AmpSlideScroll extends BaseSlides {
       );
       return false;
     }
-    showIndexArr.forEach((showIndex, loopIndex) => {
+
+    showIndexArr.sort().forEach((showIndex, loopIndex) => {
```

Comment:
super nit: I prefer `.sort()` on its own line instead of chained with forEach.  that is because it actually modified the `showIndexArr` in addition to returning it.

i.e.

```js
showIndexArry.sort();
showIndexArr.forEach(...)
```